sheetstorm-deployment/roles/configure_nginx/templates/maintenance-remote.j2

```diff
@@ -4,6 +4,15 @@ limit_conn_zone $binary_remote_addr zone=ip_connection_limit:10m;
 # Disable sending the nginx version in error pages and server header
 server_tokens off;
 
+# Don't allow the browser to render the page inside an frame
+add_header X-Frame-Options SAMEORIGIN;
+
+# Add X-Content-Type-Options header that disables concent-type sniffing on some browsers
+add_header X-Content-Type-Options nosniff;
```

Comment:
I added this also to django settings. We can test it later if it is needed in both places.